sql/scalar-functions.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: Scalar Functions
+description: 'Supported Scalar Functions in Arroyo SQL'
+---
+
+Arroyo's Scalar function support mainly mirrors that of DataFusion, 
```

Comment:
We either need an explanation of what datafusion is and our relationship to it, or leave it out if we don't think that's relevant to end-users writing queries

Reply:
Removed it from here.